crates/polars-arrow/src/bitmap/immutable.rs

```diff
@@ -146,15 +165,16 @@ impl Bitmap {
     /// # Implementation
     /// This function is `O(1)` - the number of unset bits is computed when the bitmap is
```

Comment:
We need to update this docstring. :eyes:

Reply:
Done.